go/vt/discovery/healthcheck.go

```diff
@@ -586,6 +599,13 @@ func (hc *HealthCheckImpl) updateHealth(th *TabletHealth, prevTarget *query.Targ
 	hc.broadcast(th)
 }
 
+// Recomputes the healthy tablets for the given key.
```

Comment:
Nit, but function comments should start with the function name.

Reply:
I can fix that up!

Reply:
Done!